force_bdss/base_extension_plugin.py

```diff
@@ -105,12 +127,28 @@ def __init__(self, *args, **kwargs):
             self._logger.exception(e)
             self.error_msg = str(e)
             self.error_tb = traceback.format_exc()
+            self.name = ""
```

Comment:
Would there be any difference between u"" and ""? I'd imagine not since it's just an empty string but I don't know how traits checks these things

Reply:
With the specification of unicode_literals now this string is a unicode string. I am going to introduce the same future import in the rest of the patch.